client/my-sites/customer-home/free-photo-library-card/index.jsx

```diff
@@ -35,6 +38,16 @@ const FreePhotoLibraryCard = ( { recordTracksEvent: tracks } ) => {
 		return setShowDialog( ! showDialog );
 	};
 
+	const openSupportAndTrack = () => {
+		withAnalytics(
+			recordTracksEvent( 'calypso_customer_home_free_photo_library_video_support_page_view' ),
+			supportArticleDialog( {
+				postId: 145498,
+				postUrl: 'https://support.wordpress.com/free-photo-library/',
```

Comment:
I think we still need to use a localized URL. The post ID remains the same (not sure how that is handled by the `openSupportArticleDialog` action, but we have [other examples](https://github.com/Automattic/wp-calypso/blob/0d6375eefab2179ca8d73cb1a986b4349c36f7d3/client/my-sites/site-settings/podcasting-details/support-link.jsx#L13-L14) where the post ID is fixed and the URL localized).

Reply:
Fixed! :)